index2.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles2.css">
+    <title>Concept pagina</title>
+</head>
+
+<body>
+<header class="outer-container">
+
+    <nav class="navbar">
+        <ul>
+            <li><a href="#rolling-stone-title">case studies</a></li>
+            <li><a href="#webdesign-title">services</a></li>
+            <li><a href="#footer-text">contact us</a></li>
+        </ul>
+    </nav>
+
+    <div class="hero-section">
+        <h1>Your website is an investment
+            in the succes of your business.</h1>
+        <p>We are here to help you succeed.</p>
+        <button class="cta.btn" type="button">Lets work together</button>
+    </div>
+
+
+    <img src="assets/squiggle_line_stripe.svg" alt="My Happy SVG" class="swirl">
```

Comment:
De Squiggle > je hebt nu de image tag gebruikt: voor het overzicht van de code kon de source van het bestand ook in het css bestand gekoppeld worden, tevens kon het dan van <img> een <div> worden.